cmd/pctl/show.go

```diff
@@ -39,8 +40,11 @@ func showCmd() *cli.Command {
 				return errors.New("both catalog name and profile name must be provided")
 			}
 			catalogName, profileName := parts[0], parts[1]
-
-			profile, err := catalog.Show(catalogClient, catalogName, profileName)
+			var catalogVersion string
+			if len(parts) == 3 {
+				catalogVersion = parts[2]
+			}
+			profile, err := catalog.Show(catalogClient, catalog.WithCatalogName(catalogName), catalog.WithProfileName(profileName), catalog.WithCatalogVersion(catalogVersion))
```

Comment:
we should enforce that version is provided. Nice feature down the line might be to make it optional and let pctl install the latest

Reply:
so it should be required?